mava/environment_loop.py

```diff
@@ -292,7 +292,7 @@ def step_executor() -> None:
                     results.update(eval_result)
                     self._logger.write(results)
 
-                    if self._executor.store.checkpoint_best_perf:
+                    if self._executor.store.global_config.checkpoint_best_perf:
```

Comment:
So if I understand the issue was that the previous if statement here with hasattr was never passing. Is there a default best check pointer component in the system? or Do we already have a check to make sure that whenever the best check pointer is set to true, a best check pointer component exist on the system.

Reply:
Oh nice catch @ulricharmel we missed the add of the other statement that check if the best checkpoint component exist or not.
In this if statement we need to check:
1. That the best checkpoint component exist
2. Check that even if the best checkpoint exist if the flag of checkpointing the best performance set to true

Reply:
Fixed by @sash-a